hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/pom.xml

```diff
@@ -333,6 +333,16 @@
      <dependency>
          <groupId>com.fasterxml.jackson.dataformat</groupId>
          <artifactId>jackson-dataformat-yaml</artifactId>
+         <exclusions>
+             <exclusion>
+                 <groupId>org.yaml</groupId>
+                 <artifactId>snakeyaml</artifactId>
+             </exclusion>
+         </exclusions>
+     </dependency>
+     <dependency>
+         <groupId>org.yaml</groupId>
+         <artifactId>snakeyaml</artifactId>
```

Comment:
I think instead of this exclusion, we should update `com.fasterxml.jackson.dataformat` to 2.15.2 version which uses snakeyaml 2.0.

https://mvnrepository.com/artifact/com.fasterxml.jackson.dataformat/jackson-dataformat-yaml/2.15.2

Can we try that option?

Reply:
@tomicooler Yes, but this needed to be fixed for other jackson related dependency altogether having <jackson2.version>2.12.7</jackson2.version> ,
https://github.com/apache/hadoop/blob/5aee0e0c0a2c1d4181e92a20b311885a5c552121/hadoop-project/pom.xml#L72